### DISCUSSION TITLE
How to disable Drag Selection. Currently using 7.4.0 version

### DISCUSSION THREAD

**Discussion by ganeshpatil4210** [katspaugh/wavesurfer.js#3370] [Q&A]
Tried with waveform.current.disableDragSelection(); and regions.disableDragSelection(); but unbale to do it. Is there any other approach.

**Comment by katspaugh** [comment]
enableDragSelection returns a callback. Calling it will disable drag selection. It’s in the docs: https://wavesurfer.xyz/docs/classes/plugins_regions.default#enableDragSelection
